packages/app/src/display/style/components/ui/_overlay.scss

```diff
@@ -153,6 +161,26 @@
 		width: 1em;
 		height: 0.625em;
 	}
+
+	.quick-chat-box-container {
+		display: flex;
+		flex-direction: column;
+		width: 100%;
+		align-items: center;
+		justify-content: center;
```

Comment:
Probably don't need all of these different centering methods.. find the one(s) that work, remove others.